org.adempiere.pipo/src/org/adempiere/pipo2/PoFiller.java

```diff
@@ -49,14 +50,55 @@ protected String getStringValue (String columnName){
 		return value;
 	}
 	
-	protected boolean isBlobOnPackinFile (String columnName){
-		String value = getStringValue(columnName);
-		if(value == null)
-			return false;
+	private enum TagValueType {
+		BINARY,
+		CLOB,
+		TEXT
+	}
+	
+	/**
+	 * test to see can treatment value as normal String to set to String column
+	 * 
+	 * value can be string or path to binary file
+	 * in case binary has 2 type
+	 * 1. binary is (attach file, image, binary content) then 
+	 * 		tag name = "BinaryData"
+	 * 		value by format	file path (fileCount + ".dat")
+	 * 			part 1 file name: fileCount + ".dat" // refer: PackOut.writeBlob 
+	 *      	separate char : "|"
+	 *      	part 2 data type: "byte[]" // refer:PoExporter.addBlob
+	 * 2. binary is long string (Clob) then Packout then
+	 * 		tag name is columnName
+	 * 		value by format
+	 * 			part 1 file name: fileCount + ".dat" // refer: PackOut.writeBlob 
+	 *      	separate char : "|"
+	 *      	part 2 data type: "string" // refer:PoExporter.addBlob
+	 * 3. isn't binary but value has format of case 1, 2
+	 * @param qName : xml tag name
+	 * @return
+	 */
+	protected TagValueType getTreamentType (String qName){
+		if ("BinaryData".equals(qName))
+			return TagValueType.BINARY;
 		
-		String strParts [] = value.split("[|]");
-		return strParts.length == 2;
+		Element pe = element.properties.get(qName);
+		if (pe == null) // in case binary, clob this data always not null
+			return TagValueType.TEXT;
+		
+		String value = pe.contents.toString();
+		if (StringUtils.isBlank(value)) // in case binary, clob this data always not blank

```

Comment:
is StringUtils.isBlank the same as org.compiere.util.Util.isEmpty - I think there is no need to add dependency to an apache commons library here